src/design-system/components/Box/Box.examples.tsx

```diff
@@ -129,6 +129,22 @@ export const widths: Example = {
       <Stack space="12px">
         <Box background="accent" padding="24px" width="1/3" />
         <Box background="accent" padding="24px" width="full" />
+        <Box background="accent" padding="24px" width={{ custom: 30 }} />
+      </Stack>
+    ),
+};
+
+export const heights: Example = {
+  name: 'Heights',
+  Example: () =>
+    source(
+      <Stack space="12px">
+        <Box background="accent" height="30px" padding="24px" />
```

Comment:
These padding props need to be removed, otherwise the Box can't be shorter than 48px.

Reply:
Ah derp. Sorry, that was just copy-pasta.